ocaml/xapi/smint.ml

```diff
@@ -142,6 +142,7 @@ let query_result_of_sr_driver_info x = {
 
 type attach_info = {
   params : string;
+  params_nbd: string;
```

Comment:
It may be better to define this new fields as a `string option` to better handle the cases where this (new) field has not been specified. That would avoid the `params_nbd = ""` above, where we apparently encode the don't-have-this option as an empty string, which is less explicit.

Reply:
Good idea, I will code it up.